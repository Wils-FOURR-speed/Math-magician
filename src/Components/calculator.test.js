import React from 'react';
import renderer from 'react-test-renderer';
import CalcDisplay from './calculator';

describe('CalcDisplay', () => {
  test('renders correctly', () => {
    const component = renderer.create(<CalcDisplay />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});