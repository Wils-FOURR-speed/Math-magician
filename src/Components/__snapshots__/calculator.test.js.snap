// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalcDisplay renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="welcome"
  >
    <p>
      Do your Arithmetics here. ðŸ‘‡
    </p>
  </div>
  <div
    className="calculator"
  >
    <div
      className="resultScreen"
    >
      <p>
        0
      </p>
    </div>
    <div
      className="buttonSet1"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="operatorBtn"
        onClick={[Function]}
        type="button"
      >
        Ã·
      </button>
    </div>
    <div
      className="buttonSet2"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="operatorBtn"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="buttonSet3"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="operatorBtn"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="buttonSet4"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="operatorBtn"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="buttonSet5"
    >
      <button
        className="bigger"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="operatorBtn"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
